src/Examples/Advanced/CustomProperties/ideal_h2o_co2_vle_prop_pack_testing.ipynb

```diff
@@ -0,0 +1,689 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 4 - \"state_definition\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains the particular state definition method as value, from idaes core library, according to the desired state variable combinations to be specified for it"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 5 - \"state_bounds\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains a dictionary as value, containing the state variable bounds within which the system would operate, and the property package would be valid, along with the initial points. The state variable keys in the dictionary need to be specified according to the state definition "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 6 - \"pressure_ref\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains a tuple as value, containing the systems's reference pressure, with the appropriate units"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 7 - \"temperature_ref\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains a tuple as value, containing the systems's reference temperature, with the appropriate units"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 8 - \"phases_in_equilibrium\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains a list of tuples as value, each tuple containing names of phases in equilibrium for the system"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 9 - \"phase_equilibrium_state\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains a dictionary as value, with the method required from idaes core library to obtain a smooth formulation for phase equilibrium equations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Key 10 - \"bubble_dew_method\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This key contains the method required from idaes core library to calculate bubble and dew points for the system, as a value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configuration Dictionary \n",
+    "Data Sources:\n",
+    "\n",
+    "[1] NIST Webbook, https://webbook.nist.gov/\n",
+    "\n",
+    "[2] Perry's Chemical Engineers' Handbook 7th Ed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "configuration = {\n",
```

Comment:
I think we need more description here. Given that this is basically a tutorial, I think the first part should provide a general description of the overall tool and what each part does and why you want to set it. The second part (i.e. here) should then show **how** to set each part for a given application with some justification/discussion of why/how each value was chosen.